frontend/src/app/my/mentorship/programs/[programKey]/modules/[moduleKey]/issues/page.tsx

```diff
@@ -0,0 +1,275 @@
+                  }}
+                >
+                  {l === LABEL_ALL ? 'All' : l}
+                </SelectItem>
+              ))}
+            </Select>
+          </div>
+        </div>
+
+        <div className="overflow-hidden rounded-lg border border-gray-200 dark:border-gray-700">
+          <table className="min-w-full divide-y divide-gray-200 dark:divide-gray-700">
+            <thead className="bg-gray-50 dark:bg-[#2a2e33]">
+              <tr>
+                <th
+                  scope="col"
+                  className="px-6 py-3 text-left text-xs font-medium tracking-wider text-gray-500 uppercase dark:text-gray-400"
+                >
+                  Title
+                </th>
+                <th
+                  scope="col"
+                  className="px-6 py-3 text-left text-xs font-medium tracking-wider text-gray-500 uppercase dark:text-gray-400"
+                >
+                  Status
+                </th>
+                <th
+                  scope="col"
+                  className="px-6 py-3 text-left text-xs font-medium tracking-wider text-gray-500 uppercase dark:text-gray-400"
+                >
+                  Labels
+                </th>
+                <th
+                  scope="col"
+                  className="px-6 py-3 text-left text-xs font-medium tracking-wider text-gray-500 uppercase dark:text-gray-400"
+                >
+                  Assignee
+                </th>
+              </tr>
+            </thead>
+            <tbody className="divide-y divide-gray-200 bg-white dark:divide-gray-700 dark:bg-[#1f2327]">
+              {moduleIssues.map((issue) => (
+                <tr key={issue.objectID} className="hover:bg-gray-50 dark:hover:bg-[#2a2e33]">
+                  <td className="px-6 py-4 text-sm font-medium whitespace-nowrap text-blue-600 dark:text-blue-400">
+                    <Tooltip
+                      closeDelay={100}
+                      delay={100}
+                      showArrow
+                      content={issue.title}
+                      placement="bottom"
+                      isDisabled={issue.title.length > 50 ? false : true}
+                    >
+                      <button
+                        type="button"
+                        onClick={() => handleIssueClick(Number(issue.number))}
+                        className="block max-w-md cursor-pointer truncate text-left hover:underline"
+                      >
+                        {issue.title}
+                      </button>
+                    </Tooltip>
+                  </td>
+                  <td className="px-6 py-4 text-sm whitespace-nowrap">
+                    <span
+                      className={`inline-flex items-center rounded-lg px-2 py-0.5 text-xs font-medium ${
+                        issue.state === 'open'
+                          ? 'bg-[#2cbe4e] text-white'
+                          : 'bg-[#cb2431] text-white'
+                      }`}
```

Comment:
Also I don't think these are the color that GitHub uses for these states. I wanted these to be exactly GitHub shades so that there's instant recognition of the state.

I inspected the GitHub labels for these and colors look different (GitHub shades on the right):

<img width="111" height="173" alt="Image" src="https://github.com/user-attachments/assets/55deed52-12eb-4c74-89ec-2ee37e3dc594" />
<img width="120" height="153" alt="Image" src="https://github.com/user-attachments/assets/2d2a27b4-a9ee-4fb2-9211-138fc34ba3c4" />

These would need to be updated on single Issue page too. and for Merged state I'd also prefer the GiHub shade.